books/free-programming-books-subjects.md

```diff
@@ -0,0 +1,761 @@
+* [Bookboon: IT & Programming](http://bookboon.com/en/it-programming-ebooks) Cookie required. (email address *requested*, not required)
+* [Code Catalog: A Collection of Code Examples from Prominent Open Source Projects](https://codecatalog.org) (HTML)
+* [conceptf1.blogspot.com](http://conceptf1.blogspot.com/2013/11/list-of-freely-available-programming.html)
+* [Free Smalltalk Books, collected by Stéphane Ducasse](http://stephane.ducasse.free.fr/FreeBooks.html)
+* [Free Tech Books](http://www.freetechbooks.com)
+* [Goalkicker](https://goalkicker.com) - Programming Notes for Professionals books
+* [IBM Redbooks](http://www.redbooks.ibm.com)
+* [InfoQ Minibooks](http://www.infoq.com/minibooks/)
+* [InTech: Computer and Information Science](http://www.intechopen.com/subjects/computer-and-information-science)
+* [JSBooks - directory of free javascript ebooks](https://github.com/revolunet/JSbooks)
+* [Learn X in Y minutes](https://learnxinyminutes.com)
+* [Learneroo Resources to Learn Programming](https://www.learneroo.com/modules/12/nodes/96)
+* [Microsoft Guides to Software](https://blogs.msdn.microsoft.com/mssmallbiz/2014/07/07/largest-collection-of-free-microsoft-ebooks-ever-including-windows-8-1-windows-8-windows-7-office-2013-office-365-office-2010-sharepoint-2013-dynamics-crm-powershell-exchange-server-lync-2/)
+* [Microsoft Press: Free E-Books](https://mva.microsoft.com/ebooks)
+* [Microsoft Technologies 1, including books on Windows Azure, SharePoint, Visual Studio Guide, Windows phone development, ASP.net, Office365, etc. collection by Eric Ligman](https://blogs.msdn.microsoft.com/mssmallbiz/2012/07/27/large-collection-of-free-microsoft-ebooks-for-you-including-sharepoint-visual-studio-windows-phone-windows-8-office-365-office-2010-sql-server-2012-azure-and-more/)
+* [Microsoft Technologies 2, including books on Windows Azure, SharePoint, Visual Studio Guide, Windows phone development, ASP.net, etc. collection by Eric Ligman](https://blogs.msdn.microsoft.com/mssmallbiz/2012/07/30/another-large-collection-of-free-microsoft-ebooks-and-resource-kits-for-you-including-sharepoint-2013-office-2013-office-365-duet-2-0-azure-cloud-windows-phone-lync-dynamics-crm-and-more/)
+* [O'Reilly's Open Books Project](http://www.oreilly.com/openbook/)
+* [Papers we love](https://github.com/papers-we-love/papers-we-love)
+* [Programming](https://en.wikibooks.org/wiki/Category%3aComputer_programming) - Wikibooks
+* [Red Gate Books](http://www.red-gate.com/community/books/index)
+* [Stef's Free Online Smalltalk Books](http://stephane.ducasse.free.fr/FreeBooks/)
+* [TechBeamers.com](http://www.techbeamers.com)
+* [TechBooksForFree.com](http://www.techbooksforfree.com)
+* [The Definitive C++ Book Guide and List](http://stackoverflow.com/questions/388242/the-definitive-c-book-guide-and-list)
+* [Theassayer.org](http://theassayer.org)
+* [tutorialspoint](http://www.tutorialspoint.com)
+
+
+#### Algorithms & Data Structures
```

Comment:
After moving, **heading dept should be decremented by one `###`** to be at the same level that metalist section and then, mantain the TOC synchronized :rocket:

**Apply on all headers with `####`** :wink: